big-practice/src/js/pages/dashboard.page.js

```diff
@@ -79,38 +82,43 @@ class Dashboard {
     }
   }
 
-  // /**
-  //  * Handle create form
-  //  */
-  // handleCreateForm() {
-  //   const data = {
-  //     name: nameMovieInput.value,
-  //     director: directorInput.value,
-  //     nation: nationInput.value,
-  //   };
-  //   const config = {
-  //     name: ['empty'],
-  //     director: ['empty'],
-  //     nation: ['empty'],
-  //   };
-  //   const validate = Validation.validateForm(data, config);
-
-  //   if (!validate.isValid) {
-  //     showErrorMessage(nameMovieInput, validate.errors.name);
-  //     showErrorMessage(directorInput, validate.errors.director);
-  //     showErrorMessage(nationInput, validate.errors.nation);
-  //     return;
-  //   }
-
-  //   getApi(`${MOVIES_API}?name=${data.name}`, (movieList) => {
-  //     if (movieList.length === 0) {
-  //       postApi(MOVIES_API, data, () => { renderTable(); });
-  //       hideModal(modalForm);
-  //     } else {
-  //       showErrorMessage(nameMovieInput, MESSAGES.exist);
-  //     }
-  //   });
-  // }
+  /**
+   * Handling create form  by calling API
+   */
+  async handleCreateForm() {
+    try {
+      const data = {
+        name: this.nameMovieInput.value,
+        director: this.directorInput.value,
+        nation: this.nationInput.value,
+      };
+      const config = {
+        name: ['empty'],
+        director: ['empty'],
+        nation: ['empty'],
+      };
+      const validate = formValidate.validateForm(data, config);
+
+      if (!validate.isValid) {
+        DocumentHelper.showErrorMessage(this.nameMovieInput, validate.errors.name);
```

Comment:
Could you split this code so we can use resetForm() or something to clean all form's dirty?